content/en/platform/corda/5.0/key-concepts/cordapp-dev/ecosystem/_index.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Integrating Corda into the Wider Ecosystem"
+date: 2023-07-26
+version: 'Corda 5.0'
+menu:
+  corda5:
+    identifier: corda5-cordapp-dev-ecosystem
+    parent: corda5-key-concepts-cordapp-dev
+    weight: 4000
+section_menu: corda5
+---
+
+# Integrating Corda into the Wider Ecosystem
+
+Corda is usually deployed as part of a wider ecosystem of applications in a modern enterprise. There are two possible integration points:
+* [Corda REST API]({{< relref "#corda-rest-api">}})
+* [External Messaging API]({{< relref "#external-messaging-api">}}) {{< enterprise-icon >}}
+
+Each integration point has a specific purpose.
+
+## Corda REST API
+The [Corda REST API]({{< relref "../../../reference/rest-api/_index.md">}}) is the primary way in which external applications interact with CorDapps. This API allows external systems to start flows, monitor the flow status, and retrieve the output of a flow. This REST API uses JSON for requests and responses, and flows can output data in any text representations. This gives the CorDapp Developer flexibility around how data can be consumed by external systems.
```

Comment:
`This gives the CorDapp Developer flexibility around how` ... I don't think it gives the CorDapp developer the flexibility but rather the org's flexibility to interact w/ Corda and CorDapps.